frontend/src/components/AccountReceiveTable.vue

```diff
@@ -99,12 +132,8 @@
           <div :key="props.row.id" class="col-12">
             <q-card class="card-border cursor-pointer q-pt-md col justify-center items-center">
               <q-card-section class="row justify-center items-center">
-                <img
-                  class="q-mr-md"
-                  :src="getTokenLogoUri(props.row.token, tokens)"
-                  style="width: 1.2rem"
-                  v-if="getTokenInfo(props.row.token)"
-                />
+                <img v-if="getTokenInfo(props.row.token)" src="path/to/image.jpg" />props.row.token, tokens)"
```

Comment:
is that src a mistake?

Reply:
should it be `:src="getTokenLogoUri(props.row.token, tokens)"` ?

Reply:
updated to whats in master:

https://github.com/ScopeLift/umbra-protocol/pull/685/commits/b6278897deab200b572900b6ffd6ad76fb4d8d85